graphql/util.go

```diff
@@ -1074,6 +1074,16 @@ func userHasDistroPermission(u *user.DBUser, distroId string, requiredLevel int)
 	return u.HasPermission(opts)
 }
 
+func userHasHostPermission(u *user.DBUser, distroId string, requiredLevel int, startedBy string) bool {
+	opts := gimlet.PermissionOpts{
+		Resource:      distroId,
```

Comment:
```suggestion
func userHasHostPermission(u *user.DBUser, hostId string, requiredLevel int, startedBy string) bool {
	opts := gimlet.PermissionOpts{
		Resource:      hostId,
```

Reply:
The Resource value is the distroID belonging to the host.